src/views/Drawer/DrawerView.js

```diff
@@ -123,11 +123,21 @@ export default class DrawerView<T: *> extends PureComponent<void, Props, void> {
     const DrawerScreen = this.props.router.getComponentForRouteName(
       'DrawerClose',
     );
+
+    const options = this.props.router.getScreenOptions(
+      addNavigationHelpers({
+        state: this._screenNavigationProp.state,
+        dispatch: this._screenNavigationProp.dispatch,
+      }),
```

Comment:
@satya164 Why I need to do this is mind boggling to me. 😕. I would love to have `const options = this.props.router.getScreenOptions(this._screenNavigationProp, this.props.screenProps)` which does exactly what I want but it gives me a flow error.

As an test I ran the above statement outside of `render()`, in `this. _updateScreenNavigation()`, there I get no flow error. So what's the deal? Would love to get some assistance on this. Thanks.